mlir-sycl/test/Dialect/SYCL/invalid.mlir

```diff
@@ -489,19 +471,6 @@ gpu.module @kernels {
   }
 }
 
-// expected-error @below {{'sycl.host.kernel_name' op '@kernels::@k0' does not reference a valid kernel}}
-sycl.host.kernel_name @kernel_ref -> @kernels::@k0
-
-// -----
-
-// COM: Check function is not a kernel
```

Comment:
This seems to be a rendering issue with the diff, as the test case following the one I deleted has the exact same comment and `gpu.module`.